Airports/EBOS/Ownership.txt

```diff
@@ -0,0 +1,6 @@
+SECTOR:EBOS South:0:65
```

Comment:
```suggestion
SECTOR:EBOS South:0:6500
```
According to "The Sector Subsection" of https://www.euroscope.hu/wp/ese-files-description/ - altitude limits should be in feet.